terraform/modules/ecs/logging.tf

```diff
@@ -0,0 +1,12 @@
+# TODO CLDC-2542 remove/modify this temporary logging setup
+#tfsec:ignore:aws-cloudwatch-log-group-customer-key:TODO CLDC-2542 create and encrypt with a KMS key if retaining this log group
+resource "aws_cloudwatch_log_group" "main" {

```

Comment:
Unfortunately to view logs from ecs (useful for debugging issues with the app running on it), we have to setup some cloudwatch, and then provide this to `logConfiguration` in the `aws_ecs_task_definition`